<h1>Validation FAILED.</h1><p>Build Time: 27 minutes 54 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-dynamic-data-mapping/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/com-liferay-dynamic-data-mapping/commit/17fb20981480ba743f2d463e722120eea1c3f42d">17fb20981480ba743f2d463e722120eea1c3f42d</a></p><h4>Task Summary:</h4><ul><li><strong><a href="https://test-1-20.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/139/subrepository-compile-jdk8.log">subrepository-compile-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-20.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/139/subrepository-semantic-versioning-jdk8.log">subrepository-semantic-versioning-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-20.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/139/subrepository-pmd-jdk8.log">subrepository-pmd-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-20.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/139/subrepository-source-format-jdk8.log">subrepository-source-format-jdk8</a></strong> -  :x:<pre><code>     [echo] Using proxy squid.lax.liferay.com:3128.
     [echo] Configuration on demand is an incubating feature.
     [echo] :apps:forms-and-workflow:dynamic-data-mapping:checkSourceFormatting
     [echo] /opt/dev/projects/github/liferay-portal/modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy
     [echo] /opt/dev/projects/github/liferay-portal/modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/template/BaseDDMTemplateHandler.java
     [echo] Line starts with '4' tabs, but '3' tabs are expected: ./dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/template/BaseDDMTemplateHandler.java 115
     [echo] Line starts with '5' tabs, but '4' tabs are expected: ./dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/template/BaseDDMTemplateHandler.java 121
     [echo] :apps:forms-and-workflow:dynamic-data-mapping:checkSourceFormatting FAILED
     [echo] 
     [echo] BUILD FAILED
     [echo] 
     [echo] Total time: 28.523 secs
     [echo] Gradle build finished at 2017-09-15 18:34:00.623.
     [echo] Exception in thread "main" java.lang.Exception: Line starts with '4' tabs, but '3' tabs are expected: ./dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/template/BaseDDMTemplateHandler.java 115
     [echo] Line starts with '5' tabs, but '4' tabs are expected: ./dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/template/BaseDDMTemplateHandler.java 121
     [echo] 
     [echo] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:338)
     [echo] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:215)</code></pre></li><li><strong><a href="https://test-1-20.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/139/subrepository-unit-jdk8.log">subrepository-unit-jdk8</a></strong> -  :white_check_mark:</li></ul><h5>For more details click <a href="https://test-1-1.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(master)/168/">here</a>.</h5>

--- a/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/template/BaseDDMTemplateHandler.java
+++ b/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/template/BaseDDMTemplateHandler.java
@@ -112,13 +112,13 @@ public abstract class BaseDDMTemplateHandler extends BaseTemplateHandler {
 			ddmStructureId);
 
 		List<String> fieldNames = ListUtil.fromCollection(
-				ddmStructure.getFieldNames());
+			ddmStructure.getFieldNames());
 
 		Map<String, String> nestedFields = new HashMap<>();
 
 		for (String fieldName : fieldNames) {
 			List<String> childNames = ddmStructure.getChildrenFieldNames(
-					fieldName);
+				fieldName);
 
 			for (String childName : childNames) {
 				nestedFields.put(childName, fieldName);

--- a/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy
+++ b/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy
@@ -66,7 +66,7 @@
 							<div class="option-selected option-selected-placeholder">{$strings.chooseAnOption}</div>
 						{/if}
 					{/if}
-					
+
 					<a class="select-arrow-down-container" href="javascript:;">
 						{if $selectCaretDoubleIcon}
 							{$selectCaretDoubleIcon}

